app/src/main/java/com/arctouch/codechallenge/mvp/details/DetailsScreenPresenter.java

```diff
@@ -0,0 +1,38 @@
+package com.arctouch.codechallenge.mvp.details;
+
+import com.arctouch.codechallenge.model.Movie;
+
+
+public class DetailsScreenPresenter implements DetailsScreenView {
```

Comment:
Why your presenter implements this interface?

Reply:
I have an "activity" that prints the details of the movie, and I decided to separate the responsibility of loading a movie from "storage" that at this moment is "rest", and so I could use it in any other "activity" implementing the interface that holds the "presenter" to decide when the view should load the movie. I know it could not have been the best solution, but for the time I had it was the fastest! =D

Reply:
ok 👍